R/02-filter.R

```diff
@@ -106,10 +106,12 @@ gimap_filter <- function(.data = NULL,
                                                         filter(boolVals == TRUE) %>% #drop rows where boolVals is false, so this leaves only filters that flagged a pgRNA for removal
                                                         select(id, filterName) %>% #drop the boolVals column because don't need it anymore
                                                         group_by(id) %>% #group by the pgRNA constructs
-                                                        summarize(relevantFilters = toString(filterName)), #and make a column that comma separates the relevant filters
+                                                        summarize(relevantFilters = toString(filterName)) #and make a column that comma separates the relevant filters
```

Comment:
```suggestion
                                                        summarize(relevantFilters = toString(filter_name)) #and make a column that comma separates the relevant filters
```

Reply:
If you are cool with the previous PRs review suggestion, we'll need this here too